src/main/java/Duke.java

```diff
@@ -78,6 +118,11 @@ private static void addTaskToList(String userInput) {
         }
     }
 
+    /**
+     * Updates the status of the task by marking it as done in the task list.
+     *
+     * @param taskNumberCompleted TaskNumberCompleted stores the task number which has been completed by the user.
+     */
     private static void markTaskAsDone(String userInput) {
         printLine();
         int taskNumberCompleted = Integer.parseInt(userInput.substring(userInput.indexOf(" ") + 1));
```

Comment:
Nested methods causes line to be too long. Can add in temporary variables to split it into multiple lines.

Reply:
Sure I will do it.